$MirOS: ports/lang/nwcc/patches/patch-Makefile_skel,v 1.1 2008/10/30 17:11:15 tg Exp $
--- Makefile.skel.orig	Wed Mar  4 16:11:42 2009
+++ Makefile.skel	Fri Apr 17 14:23:15 2009
@@ -107,6 +107,9 @@ install:
 test:
 	./test.sh
 
+regress:
+	./test.sh --verbose
+
 amd64_emit_gas.o: amd64_emit_gas.c amd64_emit_gas.h
 	$(CC) $(CFLAGS) amd64_emit_gas.c -c
 	
